isolates/mono_pkg.yaml

```diff
@@ -0,0 +1,10 @@
+sdk:
+  - dev
+  - stable
```

Comment:
I usually put this as an EXPLICIT SDK version – matching the lower SDK constraint in the pubspec

Reply:
Ah, makes sense. I'll update this